backend/StreamServerApp/tests/tests_videos.py

```diff
@@ -114,61 +103,14 @@ def test_movie_series_info_parsing(self):
         self.assertEqual(movie_info['type'], 'Movie')
         self.assertEqual(movie_info['title'], 'The Blues Brothers')
 
-    def test_movies_series_added_to_db(self):
-        # We check that only one Series instance is created (2 bing band theory episodes)
-        # and 4 Movie instances are created.
-        # We also check that the video fields are set correctly.
-        call_command('populatedb')
-
-        self.assertEqual(Series.objects.count(), 1)
-        self.assertEqual(Movie.objects.count(), 5)
-
-        video = Video.objects.get(name='The.Big.Bang.Theory.S05E19.HDTV.x264-LOL.mp4')
-        series = Series.objects.first()
-        self.assertEqual(video.episode, 19)
-        self.assertEqual(video.season, 5)
-        self.assertEqual(video.series, series)
-        self.assertNotEqual(video.ov_subtitle_url, "")
-        self.assertNotEqual(series.thumbnail, "")
-        self.assertEqual(os.path.isfile("/usr/src/app/Videos/folder1/The.Big.Bang.Theory.S05E19.HDTV.x264-LOL_ov.vtt"), True)
-        os.remove("/usr/src/app/Videos/folder1/The.Big.Bang.Theory.S05E19.HDTV.x264-LOL_ov.vtt")
 
-    def test_update_db(self):
-        self.assertEqual(Series.objects.count(), 0)
-        self.assertEqual(Movie.objects.count(), 0)
-        call_command('updatedb')
-        self.assertEqual(Series.objects.count(), 1)
-        self.assertEqual(Movie.objects.count(), 5)
-        shutil.copyfile("/usr/src/app/Videos/folder1/The.Big.Bang.Theory.S05E19.HDTV.x264-LOL.mp4", 
-                            "/usr/src/app/Videos/folder1/Malcolm.in.the Middle.S03E14.Cynthia's.Back.mp4")
-        call_command('updatedb')
-        self.assertEqual(Series.objects.count(), 2)
-        self.assertEqual(Movie.objects.count(), 5)
-        os.remove("/usr/src/app/Videos/folder1/Malcolm.in.the Middle.S03E14.Cynthia's.Back.mp4")
-        call_command('updatedb')
-        self.assertEqual(Series.objects.count(), 1)
-        self.assertEqual(Movie.objects.count(), 5)
 
     def test_subtitles_extraction(self):
         extract_subtitle("/usr/src/app/Videos/folder1/The.Big.Bang.Theory.S05E19.HDTV.x264-LOL.mp4",
                         "/usr/src/app/Videos/folder1/The.Big.Bang.Theory.S05E19.HDTV.x264-LOL_ov.vtt")
         self.assertEqual(os.path.isfile("/usr/src/app/Videos/folder1/The.Big.Bang.Theory.S05E19.HDTV.x264-LOL_ov.vtt"), True)
         os.remove("/usr/src/app/Videos/folder1/The.Big.Bang.Theory.S05E19.HDTV.x264-LOL_ov.vtt")
 
-    def test_subtitles_download(self):
```

Comment:
So the improvement that was pointed as future work in the doctstring ended up as deleting the test?

Reply:
You're right, actually this issue was fixed by the change in the openssl configuration, I'm gonna add the test back